### ISSUE TITLE
Faculty of computer science, Faculty of electrical engineering memorandum of cooperation for posting job ads

### ISSUE THREAD

**Issue by writerity-t-bojan** [timeriver/HumanResources#221]

**Comment by writerity-t-bojan** [comment]
@writerity-watanabe @writerityyuko I contacted with both the Deans from Faculty of computer science and Faculty of electrical engineering again about the possibility to post ads on the faculties website free of charge. While I wait for them to reply I managed to translate the memorandum of cooperation they sent it to me. It is a universal document they send to every interested company, the procedure is that we have to sign and bring back to them. Afterwards we will have the oppoortunity to post ads for job and intern position free of charge without time limit. One of the benefits are that many IT companies use these sites for ads, especially because these are all IT students and potential employees. I'm sending you the translated draft versions.

1. https://docs.google.com/document/d/1YFEbyBLVW0DjIHQxFYQSBxJUNU21IT2Dn2tfyEpvA64/edit
2. https://docs.google.com/document/d/1bRtV-MJgP3xsiNUixmm3FiVes5ii1ac40-Sc2NE0W2E/edit

https://docs.google.com/spreadsheets/d/1qdN1to2JnJPfCTtPtTiHhV9PSqHNMWf92U-pZcETBak/edit#gid=0

Please advise if it is ok, after I get a reply from the Deans to proceed with signing and delivering the memorandums to the above-mentioned faculties.

**Comment by writerityyuko** [comment]
@writerity-t-bojan 
It looks that this agreement covers a very wide range of activities and once we make an agreement, we will be able to do various activities with them.
I think it is no problem to sign this agreement.

@writerity-watanabe 
How about you ,Hide?

**Comment by writerity-watanabe** [comment]
@writerityyuko @writerity-t-bojan 
yes okay

**Comment by writerityyuko** [comment]
@writerity-t-bojan 

how is progress of this task?

**Comment by writerity-t-bojan** [comment]
@writerityyuko Hide will have to sign and then I have to take the memorandums to the both faculties, so they can sign it as well.

**Comment by writerityyuko** [comment]
@writerity-t-bojan 

you are waiting Hide's signature, are you? 
then, ok. let's wait for a while.  thanks.

**Comment by writerity-t-bojan** [comment]
@writerityyuko No, maybe we have a misunderstanding, I was waiting a reply from the deans of the faculties about how to proceed after we signed the memorandums, meanwhile they replied to me and I will prepare the documents to be sign today and tomorrow I have to take them personally to the both faculties.

**Comment by writerityyuko** [comment]
@writerity-t-bojan 

aha, ok. I leave it to you about this task.
I am interested in this activities and so asked you about status just. it is not necessary to be hurry to do it.
